src/snoowrap.js

```diff
@@ -639,6 +639,45 @@ const snoowrap = class snoowrap {
     return this._get({uri: 'api/saved_categories'}).get('categories');
   }
 
+  /**
+   * @param {string} query a query to search
+   * @param {Object} [config] config object
```

Comment:
Take inspiration from other JSDoc:
```
@param {object} options
@param {string} options.query
@param {boolean} [options.exact]
@param {boolean} [options.include_over_18]
@param {boolean} [options.include_unadvertisable]
```

Reply:
Are you sure it's best to include `query` inside the `options` object? It feels more like something separated. I think users probably wont use custom options as much so it will be kind of annoying to pass the `query` inside an object every time. What do you think?

Reply:
Since this is what we do elsewhere we should be consistent above all :)